monitoring/exporters/postgres/docker-entrypoint.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+export DATA_SOURCE_NAME="${audius_db_url}?sslmode=disable"
+
+if [ "$*" = "--read-replica" ]; then
+    export DATA_SOURCE_NAME="${audius_db_url_read_replica}?sslmode=disable"
+fi
+
+if [ "$*" = "--creator-node" ]; then
+    export DATA_SOURCE_NAME=${dbUrl}?sslmode=disable
+fi
+
+if [ "$*" = "--identity-service" ]; then
+    export DATA_SOURCE_NAME=${dbUrl}?sslmode=disable
+fi
```

Comment:
no discovery? what if creator node or identity later use a read replica

Reply:
Discovery's DATA_SOURCE_NAME is on line 3.

We can update this file at that time once we know the read replica's envar name.

Reply:
bit confused but sgtm, i trust you

Reply:
No worries! For context:

For `exporter_postgres`, we don't send a `command`: https://github.com/AudiusProject/audius-docker-compose/pull/100/files#diff-227d46fd22bb26c20dc3bc63a7eed90ad0ef1cef6c5a50fd1c6c1b97c413c649

Since we do not have a `command`, none of the `if` statements are entered since `$*` is null.

Since we don't enter any of the additional `if` statements, only the first `export DATA_SOURCE_NAME` is executed, setting it to this: https://github.com/AudiusProject/audius-docker-compose/blob/main/discovery-provider/prod.env#L5

We'll need update/rethink our if statements if/when we add read replicas to any of the other services, but punting that logic off until/if we add additional read replicas.

Reply:
sounds good, just unclear here and elsewhere that default means discovery